docs/cli.rst

```diff
@@ -169,6 +169,23 @@ directory may be loaded with:
 
     $ tmuxp load .
 
+If you try to load a config file from within a tmux session, it will ask you
+if you want to load and attach to the new session, or just load detached.
+You can also load a config file and append the windows to the current active session.
+
+::
+
+    Already inside TMUX, switch to session? yes/no
+    Or (a)ppend windows in the current active session?
+    [y/n/a]:
+
+All of these options can be preselected to skip the prompt:
+
+.. code-block:: bash
+    $ tmuxp load -y config # load attached
```

Comment:
In tmux lore, typically `-a` is reserved for attaching.

Is there any other posix / unix-like apps that have another arg they use to mean append?  (Not a requirement of course)

Reply:
I'm looking through this PR as a whole, and `-a` /  `(a)ppend` seems to make the most sense and "fit" the best to me. Nice